.vscode/settings.json

```diff
@@ -1,12 +1,7 @@
 {
-    "files.exclude": {
```

Comment:
These file exclusions are now handled in the workspace configuration since we want to exclude them for all projects